api/py/ai/chronon/scheduler/adapters/airflow_adapter.py

```diff
@@ -0,0 +1,28 @@
+from ai.chronon.scheduler.interfaces.orchestrator import WorkflowOrchestrator
+
+from airflow import DAG
+from airflow.operators.bash_operator import BashOperator
+
+
+class AirflowOrchestrator(WorkflowOrchestrator):
```

Comment:
@ezvz Are we planning to introduce Airflow as a dependency?

Reply:
We will introduce Airflow but maintain orchestrator agnosticism by abstracting flow logic. Here is why we adopt Airflow: [doc](https://docs.google.com/document/d/1WF7WNAHegJfy_OH2Uux1eKf7avLDQRiuZ-nVWMKOLeo/edit#heading=h.cz7shfyfzg0n)